includes/controls/groups/flex-container.php

```diff
@@ -0,0 +1,135 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Group_Control_Flex_Container extends Group_Control_Base {
+
+	protected static $fields;
+
+	public static function get_type() {
+		return 'flex-container';
+	}
+
+	protected function init_fields() {
+		$fields = [];
+
+		$fields['flex_direction'] = [
+			'label' => _x( 'Flex Direction', 'Flex Container Control', 'elementor' ),
+			'type' => Controls_Manager::SELECT,
+			'options' => [
+				'' => _x( 'Default', 'Flex Container Control', 'elementor' ),
+				'row' => _x( 'Row', 'Flex Container Control', 'elementor' ),
+				'row-reverse' => _x( 'Row Reverse', 'Flex Container Control', 'elementor' ),
+				'column' => _x( 'Column', 'Flex Container Control', 'elementor' ),
+				'column-reverse' => _x( 'Column Reverse', 'Flex Container Control', 'elementor' ),
+			],
+			'default' => '',
+			'selectors' => [
+				'{{SELECTOR}}' => 'flex-direction: {{VALUE}};',
```

Comment:
Try to replace all rules with variables.

Reply:
CSS variables?
If so, in which SCSS file do you think it's the most appropriate to put it?

Reply:
I mean CSS variables, not SCSS.

Reply:
For example: https://github.com/elementor/elementor-pro/blob/8fcfd360b5fbb113e0b00d83004eab6093e4aab7/modules/gallery/widgets/gallery.php#L500

Reply:
I know, already done that...
What I meant is that in order for that to work, you need to put an actual CSS somewhere, so where should I put it? `frontend.scss`? 
We need file that's applied for any type of element, since the flex controls are being used for both widgets & elements.

Reply:
OK, after some digging, looks like there is a problem.
CSS variables are inherited, which causes the child elements to inherit their parent flex properties.